src/components/base/outline-video-vimeo/outline-video-vimeo.stories.ts

```diff
@@ -0,0 +1,52 @@
+import { html, TemplateResult } from 'lit';
+import './outline-video-vimeo';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+const configuration = {
+  title: 'Content/Vimeo Video',
+  component: 'outline-video-vimeo',
+  argTypes: {
+    videoID: {
+      name: 'video-id',
+      description: 'Vimeo video id',
+      control: { type: 'text' },
+    },
+  },
+  args: {
+    videoID: `253989945`,
+  },
+  parameters: {
+    docs: {
+      description: {
+        component: `
+Vimeo video. Allows the embedded video to fit the space.
+`,
+      },
+      source: {
+        // This code sample will be used for every example unless overridden.
+        code: `
+<outline-video-vimeo
+  video-id="{{ videoID }}"
+>
+</outline-video-vimeo>
+        `,
+      },
+    },
+  },
+};
+
+export default configuration;
+
+const Template = (args = configuration.args): TemplateResult => {
+  args = {
+    ...configuration.args,
+    ...args,
```

Comment:
What other remaining args might we have here?

Reply:
@danmuzyka. Good question. :)

This general pattern is described in https://github.com/phase2/outline/discussions/109. It's what I was able to settle on to use stories in stories to avoid copy & paste and refactor challenges.

Happy to discuss more either on a call or in that thread if it's helpful.